storage/rocksdb/CMakeLists.txt

```diff
@@ -34,6 +35,13 @@ IF ((CMAKE_CXX_COMPILER_ID STREQUAL GNU) AND
   RETURN ()
 ENDIF ()
 
+# Relax unused variable warnings for release modes
```

Comment:
Why?

Reply:
This is also part of the clang 5.7 PR too: because it's in a submodule owned by facebook, and it contains unused variables in release mode. So we'll have to silence the warning until it's fixed.